src/components/DisbursementDetails/index.tsx

```diff
@@ -53,6 +61,175 @@ const initDetails: Disbursement = {
   stats: undefined,
 };
 
+enum FieldId {
+  NAME = "name",
+  REGISTRATION_CONTACT_TYPE = "registration_contact_type",
+  ASSET_CODE = "asset_code",
+  WALLET_ID = "wallet_id",
+  VERIFICATION_FIELD = "verification_field",
+}
+
+const isDisbursementValid = (inputs: Disbursement): boolean => {
+  if (!inputs.name) {
+    return false;
+  }
+
+  if (!inputs.registrationContactType) {
+    return false;
+  }
+
+  if (!inputs.asset.code) {
+    return false;
+  }
+
+  if (!hasWallet(inputs.registrationContactType)) {
+    if (!inputs.wallet.id || !inputs.verificationField) {
```

Comment:
Same here with the `wallet`.

```suggestion
    if (!inputs.wallet?.id || !inputs.verificationField) {
```